content/docs/main-concepts/api-clients.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'API Clients'
+section: 'Main Concepts'
+priority: 3
+hidden: false
+---
+
+import { Link } from 'gatsby'
+
+## What are API Clients?
+OrderCloud uses the term "API Clients" to identify various access points to your organization's data. These access points have properties that control what parties can use it, how they can gain access, and for how long that access remains valid.
+
+Think of each API client as a venue doorway; some are for staff or VIP, others are for the public, and they each have a lock or bouncer requiring the right type of key or ticket to get through. Once inside, a wristband is usually placed on the person's wrist - letting others know which door they came through.
+
+These locks are OAuth 2.0 Grant Types and they keys are the credentials needed to gain access to the data behind each API Client. The API Client used to gain access and other information about the user is encoded into an Access Token to be used on each subsequent request. This is the "wristband" that let's developers and the OrderCloud team know where and how a requestee gained access to the data.
+
+## Creating API Clients
+Like anything else on the OrderCloud platform, API Clients can be managed via our RESTful API. You may ask, "how can I create an API Client if one is required to access the API?". Whenever a new organization is created on OrderCloud, a "default" API Client is created along with a Full Access Seller user. You can use this initial context to create anything else in OrderCloud, including more API Clients.
+
+### Naming Your API Client
+Choosing the right name for your API Client is important. By picking something descriptive, future developers will better understand when they should be utilized.
+
+### Defining User Access
+You can control which parties in your organization can access each of your API Clients using any combination of the following three properties:
+
+- `AllowSeller` - All Seller Users can use the API Client
+- `AllowAnySupplier` - Supplier Users in _any_ Supplier company can use the API Client
+- `AllowAnyBuyer` - Buyer Users in _any_ Buyer company can use the API Client
+
+Creating an API Client assignment enables further control, for instance, allowing a specific Buyer company access while blocking others. Each assignment references an API Client ID paired with a Buyer or Supplier ID. `AllowAnyBuyer` and `AllowAnySeller` will nullify direct assignments for their respective company type, so be sure set those properties to `false` if you need that level of control.
```

Comment:
"In addition to the properties above, you can create API Client assignments to enable further control. For example,"